### PR TITLE
reenable broken test on solve parametric

### PR DESCRIPTION
**Pull request by dehann**
fix:
- #144

### CODE REVIEW

**Comment by dehann** [comment]
```julia
┌ Info: test solveParametric eventId
└   eventId = "01GAQAQTAHC38K44DS9YHZDKH1"
solve-testset: Error During Test at /home/dehann/.julia/dev/NavAbilitySDK/test/integration/testSolve.jl:50
  Got exception outside of a @test
  Requests did not complete in time
  Stacktrace:
    [1] error(s::String)
      @ Base ./error.jl:33
    [2] waitForCompletion(navAbilityClient::NavAbilityClient, requestIds::Vector{String}; maxSeconds::Int64, expectedStatuses::Vector{String}, exceptionMessage::String)
      @ NavAbilitySDK ~/.julia/dev/NavAbilitySDK/src/navability/services/Utils.jl:31
    [3] testSolveSessionParametric(client::NavAbilityClient, context::Client, variableLabels::Vector{String}; maxSeconds::Int64)
      @ Main ~/.julia/dev/NavAbilitySDK/test/integration/testSolve.jl:34
    [4] testSolveSessionParametric
      @ ~/.julia/dev/NavAbilitySDK/test/integration/testSolve.jl:30 [inlined]
    [5] macro expansion
      @ ~/.julia/dev/NavAbilitySDK/test/integration/testSolve.jl:58 [inlined]
    [6] macro expansion
      @ /usr/local/share/julia-1.7.3/share/julia/stdlib/v1.7/Test/src/Test.jl:1283 [inlined]
    [7] runSolveTests(client::NavAbilityClient, context::Client)
      @ Main ~/.julia/dev/NavAbilitySDK/test/integration/testSolve.jl:51
    [8] macro expansion
      @ ~/.julia/dev/NavAbilitySDK/test/integration/runtests.jl:30 [inlined]
    [9] macro expansion
      @ /usr/local/share/julia-1.7.3/share/julia/stdlib/v1.7/Test/src/Test.jl:1283 [inlined]
   [10] top-level scope
      @ ~/.julia/dev/NavAbilitySDK/test/integration/runtests.jl:20
   [11] include(fname::String)
      @ Base.MainInclude ./client.jl:451
   [12] top-level scope
      @ ~/.julia/dev/NavAbilitySDK/test/runtests.jl:2
   [13] include(fname::String)
      @ Base.MainInclude ./client.jl:451
   [14] top-level scope
      @ none:6
   [15] eval
      @ ./boot.jl:373 [inlined]
   [16] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:268
   [17] _start()
      @ Base ./client.jl:495
GraphVizApp(context) = GraphVizApp("https://app.navability.io/cloud/graph/?userId=guest@navability.io&robotStartsWith=IntegrationRobot&sessionStartsWith=TestSession2DU3m1lfg")
MapVizApp(context) = MapVizApp("https://app.navability.io/cloud/map/?userId=guest@navability.io&robotStartsWith=IntegrationRobot&sessionStartsWith=TestSession2DU3m1lfg")
```

**Comment by dehann** [comment]
@jim-hill-r , can you please take over the PR and ask @Affie for help as necessary to get this one test working please.

**Comment by jim-hill-r** [comment]
@dehann I am assigning back to you as I probably won't have much time until the weekend to look into this further and I'd like to work on unblocking you from exportSession. From here I hope you can take this one completion. I did commit changes which result in the parametric test passing. The root cause was we need to initialize the parametric key in solverDataDict. Not ideal I know, but that was the root cause. I added some code in `entities/variable.jl` to produce a passing payload.

Unfortunately this broke some of the unit tests that just check payloads, so we need to update those or comment them out. I'll leave that for you to decide. I think this may have also broken some initVariable stuff related to bounds errors so I also commented those out.

If you need me to keep working this, just assign back and I will. Just trying to help unblock all open issues as best as possible.

File: test/integration/runtests.jl [hunk 1]
```diff
@@ -8,7 +8,7 @@ include("./testFactor.jl")
 include("./testSolve.jl")
 include("./testExportSession.jl")
 
-apiUrl = get(ENV,"API_URL","https://api.navability.io")
+apiUrl = get(ENV,"API_URL","http://localhost:4343")
```
Comment (dehann) [inline test/integration/runtests.jl:11]:
```suggestion
apiUrl = get(ENV,"API_URL","https://api.navability.io")
```

**Comment by dehann** [comment]
for ref:
```julia
cloud-system-srvc-iif-worker-1        | ┌ Error: Variable does not have solver data graphinit
```

**Comment by jim-hill-r** [comment]
> for ref:
> 
> ```julia
> cloud-system-srvc-iif-worker-1        | ┌ Error: Variable does not have solver data graphinit
> ```

For ref? What does this even mean? We just caused this with the PR merge as it contained a :graphinit change from :parametric.

**Comment by dehann** [comment]
This PR is replaced by a server side fix, and a separate PR to enable the test here in SDK will be opened.

**Pull request closed without merging**